letsencrypt/cli.py

```diff
@@ -1030,6 +1030,12 @@ def renew(config, unused_plugins):
     _renew_describe_results(config, renew_successes, renew_failures,
                             renew_skipped, parse_failures)
 
+    if renew_failures or parse_failures:
+        raise errors.Error("{0} renew failure(s), {1} parse failure(s)".format(
```

Comment:
I looked around at the call chain and it seems that this error will be uncaught and ultimately cause the interpreter to exit with status code 1, but it also seems that the API would allow `renew` to return a status which could be used as the exit code -- is there a reason to prefer this over the other case?

Do we have a strategy for eventually returning different exit codes to indicate different kinds of problems?